tapir/shifts/models.py

```diff
@@ -948,6 +949,11 @@ def is_balance_positive(self):
     def get_available_solidarity_shifts(self):
         return SolidarityShift.objects.filter(is_used_up=False).count()
 
+    def get_used_solidarity_shifts_current_year(self):
+        return self.user.shift_account_entries.filter(
+            is_solidarity_used=True, date__year=timezone.now().strftime("%Y")
+        ).count()
```

Comment:
Maybe we should make the sum of the values instead of counting them? The current logic only adds entries of value 1 but it could change in the future.
There's an example of a QuerySet with a sum here : https://github.com/SuperCoopBerlin/tapir/blob/92a8aa6e80630e5c598691bfe9a13a42a783ed05/tapir/coop/models.py#L269-L271